stellar/its.js

```diff
@@ -23,6 +39,156 @@ async function removeTrustedChain(wallet, _, chain, contractConfig, arg, options
     await broadcast(operation, wallet, chain, 'Trusted Chain Removed', options);
 }
 
+async function deployInterchainToken(wallet, _, chain, contractConfig, args, options) {
+    const contract = new Contract(contractConfig.address);
+    const caller = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
```

Comment:
add a util that converts string address to ScVal address more concisely, should be useful in other places too